package.json

```diff
@@ -63,5 +63,6 @@
     "handlebars": "4.7.7",
     "node-notifier": "8.0.1",
     "eslint-plugin-import/**/json5": "^1.0.2"
```

Comment:
There was a deletion of `"eslint-plugin-import/**/json5": "^1.0.2"` in this PR. I assumed it was an accident so I removed it.